### ISSUE TITLE
Syntax errors are still shown if the highlighting level is None

### ISSUE THREAD

**Issue by lancelote** [intellij-rust/intellij-rust#7210]
## Environment

* **IntelliJ Rust plugin version:** 0.3.147.3820-211-nightly
* **Rust toolchain version:** 1.52.0-nightly (b3e19a221 2021-03-12) x86_64-apple-darwin
* **IDE name and version:** CLion 2021.1.1 (CL-211.7142.21)
* **Operating system:** macOS 10.15.7
* **Macro expansion engine:** new
* **Name resolution engine:** new

## Problem description

If the file contains syntax errors and I set the highlighting level to None the errors are still reported.

## Steps to reproduce

E.g.

```rust
fn main() {
    println!("Helloo, world!";
}
```

Select "None" in inspection widget

<img width="843" alt="Screen Shot 2021-05-06 at 22 50 42" src="https://user-images.githubusercontent.com/2500320/117357478-a974fb00-aebd-11eb-845a-db0502ec086f.png">

Ctrl+A Ctrl+X Ctrl+V sequence resolves the problem hiding the errors.
